pkgs/development/libraries/ncurses/default.nix

```diff
@@ -46,6 +46,9 @@ stdenv.mkDerivation rec {
   buildInputs = lib.optional (mouseSupport && stdenv.isLinux) gpm;
 
   preConfigure = ''
+    # These paths end up in the default lookup chain.
+    export TERMINFO_DIRS=/etc/terminfo
+
     export PKG_CONFIG_LIBDIR="$dev/lib/pkgconfig"
```

Comment:
This is taking too much attention/time.
Quoting here is correct, is resistant to future changes, and is not an issue of any kind. Plus it is not part of the modified part of this PR. Changing this will pollute git blame with no extra benefit to execution or readability.

I will not modify this PR. I guess that https://github.com/NixOS/nixpkgs/issues/26951 is a better place to discuss the general issue than here.